includes/classes/Feature.php

```diff
@@ -640,4 +641,26 @@ protected function set_settings_schema() {
 	 */
 	public function set_i18n_strings(): void {
 	}
+
+	/**
+	 * Get all features required by this feature
+	 *
+	 * @since 5.3.0
+	 * @return array List of required feature slugs
+	 */
+	public function get_required_feature() {
+		$required_feature = [];
+
+		// Add multi-feature requirements if defined
+		if ( is_array( $this->requires_feature ) ) {
+			$required_feature = array_merge( $required_feature, $this->requires_feature );
+			return array_unique( $required_feature );
+		}
+
+		if ( $this->requires_feature ) {
+			return [ $this->requires_feature ];
+		}
+
+		return [];
+	}
```

Comment:
Can't we go with a simpler solution like:

```suggestion
	public function get_required_feature() {
		return array_unique( (array) $this->requires_feature );
	}
```

Reply:
Added this in with a tiny adjustment